[Error Time: 2024-12-20 02:44:43.943 PM], [Error Level: [31merror[39m], [Error Message: 	[31mFailed to connect Payment Service Kafka Producer: Connection error: [39m], [Error Location: Unknown location]
[Error Time: 2024-12-20 02:47:00.144 PM], [Error Level: [31merror[39m], [Error Message: 	[31mFailed to send payment event [PaymentService-Kafka]: The producer is disconnected[39m], [Error Location: Unknown location]
[Error Time: 2024-12-20 12:12:52.266 PM], [Error Level: [31merror[39m], [Error Message: 	[31mFailed to connect Payment Service Kafka Producer: Connection error: connect ECONNREFUSED ::1:9092[39m], [Error Location: Unknown location]
[Error Time: 2024-12-20 12:28:07.859 PM], [Error Level: [31merror[39m], [Error Message: 	[31mFailed to connect Payment Service Kafka Producer: Connection timeout[39m], [Error Location: Unknown location]
[Error Time: 2024-12-20 04:47:40.116 PM], [Error Level: [32minfo[39m], [Error Message: 	[32mPayment Service Kafka Producer connected[39m], [Error Location: kafka.ts:21:14]
[Error Time: 2024-12-20 04:52:36.113 PM], [Error Level: [32minfo[39m], [Error Message: 	[32mPayment Service Kafka Producer connected[39m], [Error Location: kafka.ts:21:14]
[Error Time: 2024-12-20 05:05:29.172 PM], [Error Level: [32minfo[39m], [Error Message: 	[32mPayment Service Kafka Producer connected[39m], [Error Location: kafka.ts:21:14]
[Error Time: 2024-12-20 05:08:59.362 PM], [Error Level: [32minfo[39m], [Error Message: 	[32mPayment Service Kafka Producer connected[39m], [Error Location: kafka.ts:21:14]
[Error Time: 2024-12-20 05:14:20.421 PM], [Error Level: [32minfo[39m], [Error Message: 	[32mPayment Service Kafka Producer connected[39m], [Error Location: kafka.ts:26:14]
[Error Time: 2024-12-20 05:33:20.093 PM], [Error Level: [32minfo[39m], [Error Message: 	[32mPayment Service Kafka Producer connected[39m], [Error Location: kafka.ts:26:14]
[Error Time: 2024-12-20 07:47:48.478 PM], [Error Level: [32minfo[39m], [Error Message: 	[32mPayment Service Kafka Producer connected[39m], [Error Location: kafka.ts:26:14]
